Makefile

```diff
@@ -144,3 +144,25 @@ create-metrics-svc:
 .PHONY: delete-metrics-svc
 delete-metrics-svc:
 	kubectl delete -f deploy/example/metrics/csi-blob-controller-svc.yaml --ignore-not-found
+
+# compile .proto file to go output
+.PHONY: gen-proto
+gen-proto:
+	protoc --proto_path=pkg/blobfuse-proxy/proto --go-grpc_out=pkg/blobfuse-proxy/pb --go_out=pkg/blobfuse-proxy/pb pkg/blobfuse-proxy/proto/*.proto
+
+# clean the files generated from .proto file
+.PHONY: clean-proto
+clean-proto:
+	rm pkg/blobfuse-proxy/pb/*.go
+
+.PHONY: blobfuse-proxy
```

Comment:
should include `blobfuse-proxy` in ut

Reply:
done

Reply:
my fault, `unit-test` has already covered this:
```
.PHONY: unit-test
unit-test:
	go test -covermode=count -coverprofile=profile.cov ./pkg/... ./test/utils/credentials
```